### PR TITLE
fix: compile error with openssl

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: alpine-dev/Dockerfile [hunk 1]
```diff
@@ -5,13 +5,15 @@ FROM openresty/openresty:alpine-fat AS production-stage
 ARG ENABLE_PROXY
 RUN set -x \
     && (test "${ENABLE_PROXY}" != "true" || /bin/sed -i 's,http://dl-cdn.alpinelinux.org,https://mirrors.aliyun.com,g' /etc/apk/repositories) \
-    && apk add --no-cache --virtual .builddeps \
+    && apk add --no-cache --virtual .builddeps openssl-dev \
```
Comment (tokers) [inline alpine-dev/Dockerfile:8]: It should be `openresty-openssl-dev`.

File: alpine/Dockerfile [hunk 2]
```diff
@@ -8,13 +8,15 @@ LABEL apisix_version="${APISIX_VERSION}"
 ARG ENABLE_PROXY
 RUN set -x \
     && (test "${ENABLE_PROXY}" != "true" || /bin/sed -i 's,http://dl-cdn.alpinelinux.org,https://mirrors.aliyun.com,g' /etc/apk/repositories) \
-    && apk add --no-cache --virtual .builddeps \
+    && apk add --no-cache --virtual .builddeps openssl-dev \
```
Comment (tokers) [inline alpine/Dockerfile:11]: Ditto.

File: alpine-dev/Dockerfile (hunk 1 above)
Comment (gxthrj) [reply alpine-dev/Dockerfile:8]: `openresty-openssl-dev` can not be found will occur an error.

**Comment by gxthrj** [comment]
Close this PR, Repeated with #123

File: alpine-dev/Dockerfile (hunk 1 above)
Comment (tokers) [reply alpine-dev/Dockerfile:8]: See http://openresty.org/en/linux-packages.html#ubuntu for the details.

**Pull request closed without merging**